### ISSUE TITLE
Doesn't create any file on the remote end

### ISSUE THREAD

**Issue by gergoerdi** [eborisch/rpipe#1]
I'm trying to use `rpipe` in the simplest possible way, but it doesn't seem to create any file on the remote end whatsoever:

```
$ rclone purge gdrive:backup/test
$ cat a.bin |buffer|rpipe.py  gdrive:backup/test/a.bin
```
The output is:

```
Sending chunk 0 [8388608 bytes so far]
2017/04/01 00:54:11 
Transferred:      0 Bytes (0 Bytes/s)
Errors:                 0
Checks:                 0
Transferred:            0
Elapsed time:        2.6s
Sending chunk 1 [10486376 bytes so far]
2017/04/01 00:54:14 
Transferred:      0 Bytes (0 Bytes/s)
Errors:                 0
Checks:                 0
Transferred:            0
Elapsed time:        2.5s
Sending complete. Depositing metadata.
2017/04/01 00:54:16 
Transferred:      0 Bytes (0 Bytes/s)
Errors:                 0
Checks:                 0
Transferred:            0
Elapsed time:        2.5s
Final checksum checks.
2017/04/01 00:54:18 directory not found
Traceback (most recent call last):
  File "rpipe.py", line 299, in <module>
    deposit(args)
  File "rpipe.py", line 237, in deposit
    check_pipe(args.destination)
  File "rpipe.py", line 166, in check_pipe
    remote))
  File "/usr/lib/python2.7/subprocess.py", line 573, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
subprocess.CalledProcessError: Command '('rclone', 'md5sum', '--include=rp-*', '--retries=10', 'gdrive:backup/test/a.bin')' returned non-zero exit status 1
```

If I try running it with `-n`, then it doesn't complain, but nevertheless there is no `backup/test/a.bin` file created remotely.

**Comment by gergoerdi** [comment]
I should also add that I've tried it both with a Google Drive and a B2 backend, and both gives the same result.

**Comment by eborisch** [comment]
Thanks for trying it. I've only tested on ACD, so I'll have to look into the differences required with rclone for ACD vs. Google Drive or B2. My first guess (again, this was thrown together one night) is that I need to create directories first rather than having them spring into existence. I'll get back to you. I have Google Drive available, so hopefully I can reproduce this locally.

**Comment by eborisch** [comment]
I'll likely add a check at some point for what type of remote we're talking to, and automatically configure sane (e.g. md5 or not) defaults.

**Comment by eborisch** [comment]
Also, what version? 1.36 fixed some copyto bugs. (May or may not be the issue here.)

**Comment by eborisch** [comment]
I can't reproduce with gdrive destination and 1.36. Can you issue 'rclone mkdir gdrive:backup/test' and then the rpipe line? Assuming that resolves it, I'll just have it do that by default.

**Comment by eborisch** [comment]
Hopefully fixed by 62042b1d2e069443d0c32187377d8e89a0038ccf. Please let me know if it's not.

**Comment by gergoerdi** [comment]
After upgrading rclone from 1.35 to 1.36, I can see indeed that it works.
